edx_exams/apps/core/api.py

```diff
@@ -93,9 +99,43 @@ def update_attempt_status(attempt_id, to_status):
         attempt_obj.start_time = datetime.now(pytz.UTC)
         attempt_obj.allowed_time_limit_mins = _calculate_allowed_mins(attempt_obj.exam)
 
+    course_key = CourseKey.from_string(attempt_obj.exam.course_id)
+    usage_key = UsageKey.from_string(attempt_obj.exam.content_id)
+
     if to_status == ExamAttemptStatus.submitted:
```

Comment:
[question] why do we use the variable `to_status` rather than just `status`? Is this to reflect the event that the status has changed, thus triggering the event to be sent?

Reply:
Yes, exactly! to_status is implying that this is the newer status